CoreFoundation/PlugIn.subproj/CFBundle_Resources.c

```diff
@@ -27,7 +27,7 @@
 
 #if TARGET_OS_MAC || TARGET_OS_LINUX || TARGET_OS_BSD
 #include <unistd.h>
-#if !TARGET_OS_ANDROID
+#if !TARGET_OS_ANDROID || !TARGET_OS_LINUX
```

Comment:
I think `TARGET_OS_ANDROID` implies `TARGET_IS_LINUX`.

From `CoreFoundation/Base.subproj/SwiftRuntime/TargetConditionals.h`:

```
#elif __ANDROID__
#define TARGET_OS_DARWIN       0
#define TARGET_OS_LINUX        1
#define TARGET_OS_WINDOWS      0
#define TARGET_OS_BSD          0
#define TARGET_OS_ANDROID      1
#define TARGET_OS_CYGWIN       0
```

Reply:
I changed the patch to be inclusive of BSD and macOS, and exclusive of Linux/Android

Reply:
Included `CFBundle_InfoPlist.c` in the PR as it too includes sys/sysctl.h

Reply:
Hmm, I confused this with the other file (`CFBundle_InfoPlist.c`). Is the `#include` even needed here on *any* target?

Reply:
sysctl.h is still included for all Apple SDKs and is not marked as deprecated. I do not have a BSD box to verify what its status is.

Reply:
I meant that nothing in this file appears to use facilities from `<sys/sysctl.h>`, as far as I can see. There are no `sysctl*`, `KERN_*`, or `CTL_*` identifiers used.

Reply:
Ah, I understand. I see what you mean, but I wanted to keep the PR Linux-focused and not presume that it could be removed, regardless of platform.

Reply:
This change was dropped in the final merge, @tachoknight, mistake?

Reply:
Btw, I ran the tests natively on Android with both changes, ie including this one that was dropped in the final merge, and didn't have a problem.

Reply:
@buttaface Sigh, yes, a rebasing mistake. I submitted [this PR](https://github.com/apple/swift-corelibs-foundation/pull/2772) for the other file. Thanks a lot for noticing it.